cmd/minikube/cmd/start.go

```diff
@@ -158,6 +159,7 @@ func initMinikubeFlags() {
 
 	startCmd.Flags().Bool(force, false, "Force minikube to perform possibly dangerous operations")
 	startCmd.Flags().Bool(interactive, true, "Allow user prompts for more information")
+	startCmd.Flags().Bool(dryRun, false, "dry-run mode. Validates configuration, but does does not mutate system state")
```

Comment:
looks good, maybe say more explicitly what you mean by system state.
either here or in a doc on the site.